docs/odb_user/src/asciidoc/_chapters/load.adoc

```diff
@@ -637,6 +637,64 @@ odb considers the string following the “em” character as the path of the fil
 
 NOTE: odb does not load rows where the size of the input file is greater than the target database column.
 
+[[load_xml_files]]
+== Load XML Files
+Trafodion odb supports loading xml files into tables, the key construct for xml files can be an element or an attribute.
```

Comment:
XML is an acronym (eXtensible Markup Language), and so should be capitalized when it appears in prose. If it appears as a file extension (for example, xyz.xml) or as part of a code example then lower case is fine. Also, this appears to be a run-on sentence. Better perhaps would be "Trafodion odb supports loading XML files into tables. The key construct for XML files may be an element or an attribute."